File: python/examples/list_accounts.py
import objectscale_client

def main():
    endpoint = "https://10.225.108.189:443"
    username = "root"
    password = "Password123@"
    insecure = True

    try:
        client = objectscale_client.client.ManagementClient(endpoint, username, password, insecure)
        accounts = client.list_accounts()
        for account in accounts:
            print(account)
    except Exception as e:
        print("Failed to list accounts:", e)


if __name__ == '__main__':
    main()

File: examples/src/bin/list_accounts.rs
use objectscale_client::client::ManagementClient;

fn main() {
    let endpoint = "https://10.225.108.189:443";
    let username = "root";
    let password = "Password123@";
    let insecure = true;

    let mut client =
        ManagementClient::new(endpoint, username, password, insecure).expect("management client");
    let accounts = client.list_accounts().expect("list accounts");
    println!("List accounts: {:?}", accounts);
}
